src/modules/tests/test-data/cat-adi-part2/vehicle-checks/__tests__/vehicle-checks.cat-adi-part2.selector.spec.ts

```diff
@@ -1,7 +1,37 @@
 import { CatADI2UniqueTypes } from '@dvsa/mes-test-schema/categories/ADI2';
-import { vehicleChecksExist } from '../vehicle-checks.cat-adi-part2.selector';
+import {
+  getVehicleChecksDangerous,
+  getVehicleChecksSerious,
+  vehicleChecksExist,
+} from '../vehicle-checks.cat-adi-part2.selector';
 
-describe('Vehicle Checks Selector Cat ADI2', () => {
+fdescribe('Vehicle Checks Selector Cat ADI2', () => {
+
+  describe('getVehicleChecksSerious', () => {
+    it('should return false if there are no serious vehicle checks recorded', ()  => {
+      const emptySerious: CatADI2UniqueTypes.VehicleChecks = { seriousFault: false };
+      const result = getVehicleChecksSerious(emptySerious);
+      expect(result).toBeFalsy();
```

Comment:
can you change these to expect().toBe(false)etc. See https://vincenttunru.com/toBeTruthy-vs-toBe-true/

Reply:
no problem.